src/main/java/uk/ac/ebi/eva/submission/entity/SubmissionDetails.java

```diff
@@ -23,10 +23,10 @@ public class SubmissionDetails {
     @PrimaryKeyJoinColumn(name = "submission_id", referencedColumnName = "submission_id")
     private Submission submission;
 
-    @Column(nullable = false, name = "project_title")
+    @Column(nullable = false, name = "project_title", length = 500)
```

Comment:
Any way of importing the length from [SubmissionController.java](https://github.com/EBIvariation/eva-submission-ws/pull/19/files#diff-927d63922dfa25b1fc2fed5557815c1b1a748855fadd0e3edab3dc7118d7da91) ?
Or better keeping it here and importing it there ?

Reply:
updated